servicetalk-dependencies/build.gradle

```diff
@@ -32,26 +32,26 @@ dependencies {
     // Use `api` only for dependencies whose types appear in ServiceTalk API.
     api "com.fasterxml.jackson.core:jackson-databind:$jacksonVersion"
     api "com.google.api.grpc:proto-google-common-protos:$protoGoogleCommonProtosVersion"
+    api "com.google.code.findbugs:jsr305:$jsr305Version"
     api "com.google.protobuf:protobuf-java:$protobufVersion"
+    api "com.google.protobuf:protobuf-java-util:$protobufVersion"
+    api "com.sun.activation:jakarta.activation:$javaxActivationVersion"
+    api "com.sun.xml.bind:jaxb-core:$javaxJaxbCoreVersion"
     api "io.opentracing:opentracing-api:$openTracingVersion"
     api "io.zipkin.reporter2:zipkin-reporter:$zipkinReporterVersion"
     // `spi` and `core` types exposed in `log4j2-mdc-utils`
     api "org.apache.logging.log4j:log4j-core:$log4jVersion"
     api "jakarta.ws.rs:jakarta.ws.rs-api:$jaxRsVersion"
+    api "jakarta.xml.bind:jakarta.xml.bind-api:$javaxJaxbApiVersion"
     // Matchers are exposed by `servicetalk-test-resources`
     api "org.hamcrest:hamcrest:$hamcrestVersion"
+    api "org.jctools:jctools-core:$jcToolsVersion"
+    api "org.openjdk.jmh:jmh-core:$jmhCoreVersion"
     api "org.reactivestreams:reactive-streams:$reactiveStreamsVersion"
-    // Use `runtime` for dependencies which are used for implementation
```

Comment:
The Maven POM produced is identical regardless of the use of `api` or `runtime` but when imported via `platform(project(":servicetalk-dependencies"))` then `runtime` dependencies are not in scope for compile class path, ie. they are truly runtime only. Accordingly, only a few components are eligible for `runtime` status.

Reply:
Thanks for clarifying!
Consider sequencing this PR after #2218